### PR TITLE
[GitHub.io] Update disclaimers and sectioning of the preface of the intrinsics specs.

### PR DESCRIPTION
**Pull request by fpetrogalli**

### CODE REVIEW

File: main/acle.md [hunk 1]
```diff
@@ -15,6 +15,13 @@ This token is necessary to generate the Table of Contents via Jekyll. --->
 <!---END_OF_HTML_HEADER--->
 # Preface
 
+## Warning
+
+The specifications in this document are presented via this website in
```
Comment (ValeriaDaneva) [inline main/acle.md:20]: This phrasing is a bit clunky I find.

```suggestion
The specifications documented on this website are in experimental format.
```

Comment (fpetrogalli) [reply main/acle.md]: Done

**Comment by ValeriaDaneva** [comment]
I approve of these changes.

**Review by ValeriaDaneva — COMMENTED** [review]
I approve of the changes.

**Review by ValeriaDaneva — APPROVED** [review]

**Comment by fpetrogalli** [comment]
@allcontributors please add @ValeriaDaneva for review.

**Pull request merged**